src/APIGateway/APIGateway/ocelot.json

```diff
@@ -0,0 +1,79 @@
+{
+  "Routes": [
+    {
+      "DownstreamPathTemplate": "/items",
+      "DownstreamScheme": "http",
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "localhost",
+          "Port": 5001
+        }
+      ],
+      "UpstreamPathTemplate": "/api/catalog/items",
+      "UpstreamHttpMethod": [ "GET" ],
+      "Key": "Items"
+    },
+    {
+      "DownstreamPathTemplate": "/items/{id}/properties",
+      "DownstreamScheme": "http",
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "localhost",
+          "Port": 5001
+        }
+      ],
+      "UpstreamPathTemplate": "/api/catalog/items/{id}/properties",
+      "UpstreamHttpMethod": [ "GET" ],
+      "Key": "ItemProperties"
+    },
+    {
+      "DownstreamPathTemplate": "/{everything}",
+      "DownstreamScheme": "http",
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "localhost",
+          "Port": 5001
+        }
+      ],
+      "FileCacheOptions": {
+        "TtlSeconds": 60
+      },
+      "UpstreamPathTemplate": "/api/catalog/{everything}",
+      "UpstreamHttpMethod": [ "GET" ]
+    },
+    {
+      "DownstreamPathTemplate": "/{everything}",
+      "DownstreamScheme": "http",
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "localhost",
+          "Port": 5001
+        }
+      ],
+      "UpstreamPathTemplate": "/api/catalog/{everything}",
+      "UpstreamHttpMethod": [ "POST", "DELETE", "PATCH" ]
+    },
+    {
+      "DownstreamPathTemplate": "/{everything}",
+      "DownstreamScheme": "http",
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "localhost",
+          "Port": 5002
+        }
+      ],
+      "UpstreamPathTemplate": "/api/carts/{everything}",
+      "UpstreamHttpMethod": [ "GET", "POST", "DELETE", "PATCH" ]
+    }
+  ],
+  "Aggregates": [
+    {
+      "RouteKeys": [
+        "Items",
+        "ItemProperties"
+      ],
+      "UpstreamPathTemplate": "/api/catalog/detailed-items",
+      "Aggregator": "CatalogItemAggregator"
+    }
+  ]
+}

```

Comment:
Security. All Endpoints from Catalog service which mutates data (POST/PUT/PATCH/DELETE) should be accessible by the admins `(Role=admin)` only. Use JWT Bearer type of authentication for this.